Makefile

```diff
@@ -58,7 +63,12 @@ CALYPSO_ENV ?= $(NODE_ENV)
 
 export NODE_ENV := $(NODE_ENV)
 export CALYPSO_ENV := $(CALYPSO_ENV)
-export NODE_PATH := server:client:.
+export NODE_PATH := server$(SEPARATOR)client$(SEPARATOR).
+ifeq ($(OS),Windows_NT)
+export NPM_GLOBAL_ROOT := $(shell $(NPM) -g root)
```

Comment:
What does `npm -g root` return on Windows that makes this necessary?  Worth documenting in a comment?